docs/markdown/testing/develop.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Development
+component: false
+---
+
+## Set up your laptop
+
+- Install a code editor - we recommended **[VS Code](https://code.visualstudio.com/download)**
+- **Install nvm**
```

Comment:
1. how are we setting the links here? 
Also I see different link behaviors
![Kapture 2022-07-26 at 13 35 45](https://user-images.githubusercontent.com/10593890/181073623-46c7bebd-57bf-4e63-9029-169cd35ffa3a.gif)